Resources/Maps/_RNMC14/Shuttles/akademia_nauk.yml

```
@@ -143,94 +142,92 @@ entities:
         params:
           volume: -1
         path: /Audio/_RMC14/Dropship/dropship_incoming.ogg
-- proto: CMAirlockSecure
+- proto: CMAkademiaDoorAft
```

You can actually go through the map file and rename the proto id to a new one, so if `CMAkademiaDoorAft` becomes `RNMCAkademiaDoorAft`, no errors occur.
```suggestion
- proto: RNMCAkademiaDoorAft
```